proto/cacheclient.proto

```diff
@@ -139,6 +139,16 @@ message _DictionarySetRequest {
 
 message _DictionarySetResponse {}
 
+message _IncrementRequest {
+  bytes cache_key = 1;
+  int64 increment_by = 2;
+  uint64 ttl_milliseconds = 3;
```

Comment:
originally I was going to suggest here to add a `bool refresh_ttl = 4;`, but thinking about it more, I dont think it makes sense for this. Because inside of our backend, this is essentially a special unary operation, and for unary sets, we also don't have a refresh ttl semantic. So I think this looks good 👍

Reply:
Yeah, I had thought about it too and also settled on not having it since this is a basically "unary" operation and `refresh_ttl` is for when you have an action that handles part of a datastructure where the overall datastructure has a ttl (more or less). I'll add a comment to the commit message though about this.